help/compute/compute.md

```diff
@@ -443,12 +443,33 @@ Automatic outliers detection Manual outliers markup and annotation Used as an in
 
 ### Sensitivity analysis
 
-* Sample inputs:
-  * by specified number of samples
-  * by a specified distribution or within a range
-  * for a specified set of scalar inputs and/or columns of the matrix input
-* Produce variability analysis for outputs based on the sampled inputs
-* Visualize the results of analysis with Datagrok viewers
+Study what happens to a function when varying its inputs. Datagrok provides investigation of how uncertainty in its output can be divided to different sources of uncertainty in the inputs.
```

Comment:
Simplify the message. Something like that (wrote it in a minute, probably can do better)

"Sensitivity Analysis runs the computation multiple times with varying inputs, and analyzes how results are dependent on parameters.

Reply:
Done